### PR TITLE
B/7208 revert temp

### PR DESCRIPTION
**Pull request by abp6318**
1. Instructions for testing:
- No location (never been saved in workspace) -- http://localhost:3333/html/arcgis-hub-workspace/entity-workspace-harness.html?identifier=5274dc3dff494b3ea1ac7424d29a9fe9&type=content
- Custom location (never been saved in workspace) -- http://localhost:3333/html/arcgis-hub-workspace/entity-workspace-harness.html?identifier=a76e7f5bcf7a4f75904ffbd51f3963cf&type=content
- Custom location (has been saved in workspace) -- http://localhost:3333/html/arcgis-hub-workspace/entity-workspace-harness.html?identifier=e22404f554fb455aa8b53c30ea7fc7f8&type=content
- Custom location not set (never been saved in workspace) -- http://localhost:3333/html/arcgis-hub-workspace/entity-workspace-harness.html?identifier=3c5e02dc7f684ba383fc79c28df0e6b0&type=content

1. Closes Issues: #7208 

1. [ ] Updated meaningful TSDoc to methods including Parameters and Returns, see [Documentation Guide](https://esri.github.io/hub-components/storybook/?path=/story/guides-documentation--page)

1. [x] used semantic commit messages
  
1. [x] PR title follows semantic commit format (**CRITICAL** if the title is not in a semantic format, the release automation will not run!)

1. [ ] updated `peerDependencies` as needed. **CRITICAL** our automated release system can **not** be counted on to update `peerDependencies` so we _must_ do it manually in our PRs when needed. See the [updating peerDependencies](/RELEASE.md#Updating-peerDependencies) section of the release instructions for more details.

### CODE REVIEW

**Comment by sonofflynn89** [comment]
@abp6318 for my edification, what's the context for this PR? How come you had to revert changes?

File: packages/common/src/content/_internal/computeProps.ts [hunk 1]
```diff
@@ -17,5 +42,13 @@ export function computeProps(
...
   content.thumbnailUrl = getItemThumbnailUrl(model.item, requestOptions, token);
 
+  if (!content.location) {
+    // build location if one does not exist based off of the boundary and the item's extent
+    content.location =
+      model.item.properties?.boundary === "none"
+        ? { type: "none" }
+        : deriveLocationFromItemExtent(model.item.extent);
+  }
+
```
Comment (tomwayson) [inline packages/common/src/content/_internal/computeProps.ts:52]: This logic will soon need to be updated for https://devtopia.esri.com/dc/hub/issues/7527#issuecomment-4269838

File: packages/common/test/content/computeProps.test.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        properties: {
+          hub: {
+            enabled: true,
+          },
+        },
+      } as unknown as IPortal,
+      portalUrl: "https://org.maps.arcgis.com",
+    });
+  });
+  it("getItemExtent isBBox is true", () => {
```
Comment (tomwayson) [inline packages/common/test/content/computeProps.test.ts:37]: you should move this into it's own describe - you don't need the context manager that is set up in the `befoeEach()`

File: packages/common/test/content/computeProps.test.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+      [100, 100],
+      [120, 120],
+    ]);
+    expect(chk.xmin).toBe(100);
+    expect(chk.ymin).toBe(100);
+    expect(chk.xmax).toBe(120);
+    expect(chk.ymax).toBe(120);
+  });
+
+  it("deriveLocationFromItemExtent valid extent", () => {
```
Comment (tomwayson) [inline packages/common/test/content/computeProps.test.ts:48]: Same here

**Review by tomwayson — APPROVED** [review]
good work

I wonder if there's any harm in merging this knowing that we're about to change the logic in https://devtopia.esri.com/dc/hub/issues/7527

**Pull request merged**